src/spaceObjects/blackHole.cpp

```diff
@@ -65,6 +65,10 @@ void BlackHole::collide(Collisionable* target, float collision_force)
     if (update_delta == 0.0)
         return;
 
+    P<SpaceObject> obj = P<Collisionable>(target);
```

Comment:
@daid As far as I know all Collisionables inside EE are also SpaceObjects, right?

Reply:
Yes, but still, checking is good practice and very cheap.

Reply:
I'm always checking. :laughing: 

I was asking, because with returning early any non-spaceObject Collisionable would not be moved as before. I could have made this clearer in the question.

Reply:
If it is a spaceobject, we do not really care about it now, as no wait to check if it has weight :)